examples/nlp/multi_label_classification.py

```diff
@@ -370,10 +363,6 @@ def plot_result(item):
 _, categorical_acc = shallow_mlp_model.evaluate(test_dataset)
 print(f"Categorical accuracy on the test set: {round(categorical_acc * 100, 2)}%.")
 
-"""
-The trained model gives us an evaluation accuracy of ~87%.
```

Comment:
This seems like a helpful line, why remove it?

Reply:
Added back in.